### DISCUSSION TITLE
Is there a way to use show just month and year level for mantine v5.

### DISCUSSION THREAD

**Discussion by Juanryhn** [mantinedev/mantine#5602] [Q&A] [answered]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

5

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

 I tried upgrade mantine to latest version but I got errors. I use react v18 and next v12. it says createStyles is not a function.
 "_mantine_core__WEBPACK_IMPORTED_MODULE_1__.createStyles) is not a function" and "Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined."

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
No, it is not supported, this feature is available only starting from version 6.0

**Reply by Juanryhn** [reply]
But can you help me why when I upgrade mantine to the latest version I got some errors. Like this "mantine_core__WEBPACK_IMPORTED_MODULE_1_.createStyles) is not a function" and "Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined."

**Reply by rtivital** [reply]
1. https://mantine.dev/guides/6x-to-7x/
2. https://mantine.dev/changelog/7-0-0/

**Reply by Juanryhn** [reply, marked as answer]
I solve this issue by upgrading just mantine core, dates, dropzone and carousel not all. from v5 to v6.0.21. Thanks @rtivital
